src/iconv.jl

```diff
@@ -1,9 +1,77 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 module iconv
-import Base: close, eof, flush, read, readall, write
+import Base: close, eof, flush, read, readall, write, show
 import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ
 export StringEncoder, StringDecoder, encode, decode
+export StringEncodingError, GeneralError
+export InvalidEncodingError, InvalidSequenceError, IncompleteSequenceError, IConvError
+
+
+abstract StringEncodingError
+
+# For now, don't display module name, maybe want it if this changes to StringEncoders
+const modulename = ""
```

Comment:
Remove this for now (and the `mod` field of exceptions), we can always add it later if needed.